internal/luks2/cryptsetup_test.go

```diff
@@ -210,6 +210,18 @@ func (s *cryptsetupSuite) TestFormatWithCustomKeyslotsAreaSize(c *C) {
 			KeyslotsAreaKiBSize: 2 * 1024}})
 }
 
+func (s *cryptsetupSuite) TestFormatWithInlineCryptoEngine(c *C) {
+	key := make([]byte, 32)
+	rand.Read(key)
+	s.testFormat(c, &testFormatData{
+		label: "test",
+		key:   key,
+		options: &FormatOptions{
+			KDFOptions:          KDFOptions{MemoryKiB: 32 * 1024, ForceIterations: 4},
+			KeyslotsAreaKiBSize: 2 * 1024,
+			InlineCryptoEngine:  true}})
+}
```

Comment:
As this test fails in the github workflow and this branch doesn't contain the feature detection parts that would allow it to be skipped, it might be better to just mock this specific test just to verify that the expected commandline arguments are supplied.

Reply:
I pushed a fix for this now